pkg/geo/wkt/wkt_test.go

```diff
@@ -98,11 +141,22 @@ func TestUnmarshalError(t *testing.T) {
 			input:          "POINT(2 2.3.7)",
 			expectedErrStr: "lex error: invalid number at pos 8",
 		},
+		// ParseError
 		{
 			desc:           "2D point with extra comma",
 			input:          "POINT(0, 0)",
 			expectedErrStr: `parse error: could not parse "POINT(0, 0)"`,
 		},
+		{
+			desc:           "2D linestring with only one point",
+			input:          "LINESTRING(0 0)",
+			expectedErrStr: `parse error: could not parse "LINESTRING(0 0)"`,
+		},
+		{
+			desc:           "linestring with mixed dimensionality",
+			input:          "LINESTRING(0 0, 1 1 1)",
+			expectedErrStr: `parse error: could not parse "LINESTRING(0 0, 1 1 1)"`,
```

Comment:
can we return the position in the error message here?

the unfortunate side effect of doing it the way we're doing it now is we cannot give a great error message when there's mixed dimensionality but should be able to somehow tell this to the user. maybe worth a TODO.